Convert all source/**/*.html to .md

levitte opened the pull request:
Partial fix of #324

levitte committed:
Convert all source/*.html to .md

Partial fix of #324

levitte committed:
Convert all source/old/*.html to .md

Partial fix of #324

t8m commented on source/index.md:
```diff
@@ -0,0 +1,64 @@
+  </tr>
+  <!--#include virtual="index.inc" -->
+</table>
+
```
Somehow the table is merged with the next paragraph in the https://beta.openssl.org/source/ which looks weird.

t8m commented on source/gitrepo.md:
```diff
@@ -0,0 +1,41 @@
+    $ cd openssl
+    $ git config core.autocrlf false
+    $ git config core.eol lf
+    $ git checkout .
```
This (and the shell code above) does not seem to be properly rendered on https://beta.openssl.org/source/gitrepo.html

levitte replied:
Okie

t8m commented on source/license.md:
```diff
@@ -0,0 +1,21 @@
+---
+title: License
+---
+# License
```
Extra # here

t8m replied:
Also on beta.openssl.org the title is duplicated.

levitte replied:
Ah, it was a bad idea to use `title:`, as this is understood by pandoc.  I've just submitted #350, which uses `breadcrumb` instead, which is actually what it's used for...  I'll have to adjust all the pages now...

levitte replied:
I hope I've fixed it, together with #350.

t8m replied:
There is still the extra `#`

t8m replied:
Or you need to drop the `--------` under `# License`

levitte replied:
Oh!  Yeah

levitte replied:
Done

t8m commented on source/mirror.md:
```diff
@@ -0,0 +1,19 @@
+| HU     | <ftp://ftp.kfki.hu/pub/packages/security/openssl/> |
+| PL     | <ftp://guest.kuria.katowice.pl/pub/openssl/>       |
+| PL     | <rsync://ftp.tpnet.pl/pub/security/openssl/>       |
+
```
For some reason on beta.openssl.org the table is merged with the paragraph after it. Perhaps the md->html conversion should handle this?

t8m replied:
Also the table styling has some glitches.

levitte replied:
There's a lot of weirdness happening with tables...  it's not in the generation, that looks perfectly fine, so it's something else.  In the HTML, this was solved with an added `<p>&nbsp;</p>` after the table, which is indeed an ugly workaround...

Re the rest of the glitches, I'll adjust that tomorrow

levitte replied:
It seems that surrounding any table with \<p>...\</p> fixes the problem

t8m replied:
It was resolved for the downloads table but not for the table with the list of the mirrors. Please look at https://beta.openssl.org/source/mirror.html

levitte replied:
Fixed

levitte commented:
I discovered that the inclusion of legalities.shtml was inadvertently dropped. Fixed

levitte committed:
$(SRCLISTS) isn't phony!

This was probably around because we used this variable to list tarballs.
Things have changed since, and making the contents of this variable phony
is now in error.

levitte commented:
All major glitches gone.  Final review, @openssl/omc?

levitte commented:
Ping

levitte commented:
Rebasee

levitte commented:
Rebased and squashed

levitte commented:
Ping

levitte commented:
Merged

239296da4d2d17929d7b5285b749cdeef3e44ad6 Convert all source/*.html to .md
fe6c01f3d893a28ccadd090f3908fe11b7dd6d3f Convert all source/old/*.html to .md
a18e42645868ebce68957fc86cdcafe613016931 $(SRCLISTS) isn't phony!
